roles/edpm_multipathd/tasks/install.yml

```diff
@@ -30,21 +30,31 @@
     - name: Gather services facts
       ansible.builtin.service_facts:
 
-    - name: Stop multipathd service on the host
-      ansible.builtin.systemd:
-        name: "{{ item }}"
-        state: stopped
-        enabled: false
+    - name: Stop multipathd on the host and restart edpm_multpathd if running
```

Comment:
I'd prefer to keep the original name. The need to restart edpm_multipathd is a corner case created by the original bug (which your other patch fixed), and it looks odd to talk about restarting a service in the tasks that are meant to install it.